### DISCUSSION TITLE
[Projects Beta] Workflows move item to top of column

### DISCUSSION THREAD

**Discussion by csprle** [community/community#7765] [Projects and Issues]
When an issue in the new projects is moved to done automatically (for example) is gets moved to the bottom of the list. It would be nice to have an option where to move issues to. 
We use the Done column to see what was done since we last had a view onto the board. When issues are at the bottom of the list, it is hard to see what was happening.

**Comment by cyager7** [comment]
> When an issue in the new projects is moved to done automatically (for example) is gets moved to the bottom of the list. It would be nice to have an option where to move issues to. We use the Done column to see what was done since we last had a view onto the board. When issues are at the bottom of the list, it is hard to see what was happening.

Is there any update on this?
